tools/scenarios-plugin.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
hm, this isn't a ts file so why are we removing the `'use strict';`? Did we lose a linter rule to enforce this?

Reply:
It's because its falling under the settings of the root eslintrc which disallows 'use strict'; vs in the test subdirectory the eslint config requires it. I brought over the config.